app/mailboxes/support_mailbox.rb

```diff
@@ -58,12 +58,15 @@ def create_conversation
   end
 
   def find_or_create_contact
-    @contact = @inbox.contacts.find_by(email: processed_mail.from.first)
+    # changing to downcase to avoid case mismatch while finding
+    @contact = @inbox.contacts.find_by(email: processed_mail.from.first.downcase)
```

Comment:
We would also need to downcase the email passed into the contact builder in 
https://github.com/chatwoot/chatwoot/blob/a7ca55c0801561bbd0c4a9e21a775569914a7d6c/app/mailboxes/support_mailbox.rb#L75

Since we are repeating the `.downcase` method, another approach could be implementing a method inside  `chatwoot/app/presenters/mail_presenter.rb ` which downcases the from addresses by default so that we wouldn't need to repeat calling downcase 

example
```
def from
   @mail.from.map(&:downcase)
end
```
ref: https://ruby-doc.org/stdlib-2.5.1/libdoc/delegate/rdoc/SimpleDelegator.html

----

let's also downcase the email used for querying here
https://github.com/chatwoot/chatwoot/blob/a7ca55c0801561bbd0c4a9e21a775569914a7d6c/app/builders/contact_builder.rb#L45